docs/COMPILATION.MD

```diff
@@ -26,6 +26,53 @@ sudo apt-get install -y libleptonica-dev
 
 2. Compiling
 
+**[Recommended] Using the build script with OCR**
+
+
+```
+#Navigate to linux directory and call the build script
+
+cd ccextractor/linux
+./build -DWITH_OCR=ON
+
+# test your build
+./ccextractor
+```
+
+**[Recommended] Standard linux compilation through Autoconf scripts with OCR**
+
+```
+sudo apt-get install autoconf      #Dependency to generate configuration script
+cd ccextractor/linux
```

Comment:
Why seperate OCR and Non OCR? Why not something like this?

```Using build.command script:

cd ccextractor/mac
./build.command OCR

# test your build
./ccextractor

If you don't want the OCR capabilities, then you don't need to configure the tesseract and leptonica packages, and build it with just

cd ccextractor/mac
./build.command

# test your build
./ccextractor
```

Reply:
@MatejMecka I don't think that makes much of a difference. In the linux one, it is for multiple options and hence I have included it once instead of including it in each of the options. I feel that's a better way, and that way we can also use the recommended tag. What do you say?

Reply:
I don't know about the Recommended one :/ Ask someone else for that. But i think it would be nicer like the method I mentioned. Because Right now you have OCR,OCR,OCR and then Normal Compilation, Normal Compilation, Normal Compilation. Normal Compilation then OCR sounds much nicer. So yours should be Normal Compilation, OCR, Normal Compilation, OCR like that.

Reply:
I think you can get a same method for first part (like navigate to ccextractor/linux, etc), you can wrap them up into one thing